analytics/pubstack/eventchannel/eventchannel_test.go

```diff
@@ -171,5 +175,34 @@ func TestEventChannel_OutputFormat(t *testing.T) {
 	expected := append(toGzip("one"), toGzip("twothree")...)
 
 	assert.Equal(t, expected, data)
+}
+
+func TestEventChannelPushAndFlush(t *testing.T) {
+	data := make([]byte, 0)
+	send := newSender(&data)
+
+	eventChannel := NewEventChannel(send, maxByteSize, maxEventCount, maxTime)
+
+	eventChannel.Push([]byte("one"))
+	eventChannel.Push([]byte("two"))
+	eventChannel.flush()
+
+	time.Sleep(10 * time.Millisecond)
```

Comment:
Could we change this to 1 ms? If we need to sleep due wait for go routines, IMHO it should be a minimal amount of time.